example/deploy/role.yaml

```diff
@@ -171,31 +171,76 @@ metadata:
   creationTimestamp: null
   name: network-operator
 rules:
-- apiGroups:
-  - mellanox.com
-  resources:
-  - '*'
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
+  - apiGroups:
+    - rbac.authorization.k8s.io
+    resources:
+    - roles
+    - rolebindings
+    verbs:
+    - '*'
+  - apiGroups:
```

Comment:
id like to keep cluster scope resource access to a minimum as well as the verbs the operator is allowed to perform on them.

can you give a bit more context why these new additions are needed ?

Reply:
@adrianchiris Are you asking specifically about this `"" apiGroup` or in general about all the cluster scoped resources?

Reply:
in general, sorry i should have placed this comment on top at L#174